src/components/Worker.tsx

```diff
@@ -0,0 +1,111 @@
+                    aria-hidden="true"
+                >
+                    <div className="modal-dialog">
+                        <div className="modal-content modal-popup">
+                            <a href="#" className="close-link"><i className="icon_close_alt2"></i></a>
+                            <img src={workerInfo.imageUrl} alt="Team Image" className="shit" />
+                            <b>
+                                <h3>
+                                    <center>{workerInfo.firstName + " " + workerInfo.lastName}</center>
+                                </h3>
+                            </b>
+                            <b>
+                                <p><center>{workerInfo.title}</center></p>
+                            </b>
+                            <b>
+                                <p><center>{age} years old</center></p>
+                            </b>
+                            <b>
+                                <p><center>{workerInfo.time}</center></p>
+                            </b>
+                            <b>
+                                <p><center>Timezone: {workerInfo.city}</center></p>
+                            </b>
+                            <hr />
+                            <div className="bio">
+                                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent
+                                vel vehicula nisl. Ut mauris metus, fermentum nec orci nec,
+                                sagittis laoreet eros. Nam porta luctus mi, sodales aliquam nibh
+                                convallis ac. Nunc tempor malesuada lobortis. Pellentesque
+                                habitant morbi tristique senectus et netus et malesuada fames ac
+                                turpis egestas. Phasellus eu tristique enim. Sed id malesuada
+                                velit. Suspendisse ut nisi quis neque venenatis vestibulum id non
+                                ipsum. Aliquam aliquet dapibus nisi, ut tempor massa hendrerit ut.
+                                Sed efficitur ex posuere, interdum turpis a, maximus urna. Aenean
+                                sed sapien nec erat viverra lobortis cursus sit amet ex. Maecenas
+                                vel tempus mi, volutpat maximus tortor. Suspendisse tempus eget
+                                ligula id mattis. Nam porttitor mauris a accumsan convallis.
+                            </div>
+                            <br />
+                            <ul className="list-inline social-buttons">
+                                <li>
+                                    <a href="https://facebook.com" target="_blank">
+                                        <img
+                                            src="../../img/img/facebook.png"
+                                            className="icon"
+                                            height="40px"
```

Comment:
you know we don't like to use pixels in the styling of out components.
Add a classname and change it to get its height and width from the CSS file and not from the code itself